package.json

```
@@ -38,13 +39,26 @@
     "eslint-plugin-react-hooks": "^4.1.2",
     "eslint-plugin-react-native": "^3.9.1",
     "husky": "^4.2.3",
+    "jest": "^26.4.2",
     "prettier": "^2.1.2",
     "react": "16.11.0",
     "react-native": "0.62.2",
+    "react-test-renderer": "^16.13.1",
     "release-it": "^13.5.1"
   },
+  "jest": {
+    "preset": "react-native",
+    "modulePathIgnorePatterns": [
+      "<rootDir>/examples/",
+      "<rootDir>/lib/"
+    ],
+    "setupFilesAfterEnv": [
+      "./jest/setupTests.js"
+    ]
+  },
   "husky": {
     "hooks": {
+      "pre-push": "yarn test && yarn lint",
```

```suggestion
      "pre-push": "yarn prepush",
```
please add separate command

---

done